package.json

```diff
@@ -408,13 +497,12 @@
     "vinyl-sourcemaps-apply": "^0.2.1",
     "watchify": "^4.0.0",
     "webextension-polyfill": "^0.8.0",
-    "webpack": "^5.75.0",
-    "yargs": "^17.0.1",
-    "yarn-deduplicate": "^3.1.0"
+    "webpack": "^4.41.6",
```

Comment:
Looks like webpack has been downgraded here. Presumably this was not on purpose?

Reply:
Thanks for catching this, looks like it was a rebase issue with recent-ish changes. Resolved in latest.